streaming/src/main/scala/org/apache/spark/streaming/scheduler/JobScheduler.scala

```diff
@@ -126,6 +126,8 @@ class JobScheduler(val ssc: StreamingContext) extends Logging {
     eventLoop.post(ErrorReported(msg, e))
   }
 
+  def isStarted(): Boolean = eventLoop != null
```

Comment:
It's better to add `synchronized` here. Although the test only uses one thread now, someone may use it in multiple threads in future.